File: geocode/geocode.js
function GEOCODE(address){
  var r = Maps.newGeocoder().geocode(address)
        for (var i = 0; i < r.results.length; i++) {
            var res = r.results[i]
            return res.geometry.location.lat + ", " + res.geometry.location.lng
  
  }
}

function RGEOCODE(latlong){
  var latlong = latlong.split(',')
  var lat = latlong[0];
  var long = latlong[1];
  var response = Maps.newGeocoder().reverseGeocode(lat, long);
  var address = response["results"][0].formatted_address;
  return address
}

File: geocode/geocode.py
import requests

api_key = '' #YOUR_API_KEY

def geocode(address):
  params = {'address':address,'key':api_key}
  response = requests.get('https://maps.googleapis.com/maps/api/geocode/json', params=params)
  return response.text
  location = response['results'][0]['geometry']['location']
  lat = location['lat']
  lng = location['lng']
  delimiter = ','
  return delimiter.join([lat,lng])

def rgeocode(lat,lng):
  response = {}
  address = response['results'][0].formatted_address
  return address